syntax = "proto3";

package ficus;

message GrpcGuid {
    string guid = 1;
}